backend/src/controller/Boards/Boards.controller.js

```diff
@@ -0,0 +1,31 @@
+import { Router } from 'express';
+import { BoardsService } from '../../service/Boards/BoardsService.js';
+import { Board } from '../../service/Boards/Board.js';
+
+export function BoardsController() {
+  const router = Router();
+  router
+    .route('/')
```

Comment:
```suggestion
    .route('/boards')
```

Reply:
ale, jeśli wprowadzę te zmiany wyżej (w app.js) to zapytanie chyba będzie musiało wyglądać tak: /rest-api/boards/boards?

Reply:
Dałem poprawkę powyżej, bo chyba się pomyliliśmy :)

Reply:
Uczywiście chodzi nam o `/rest-api/boards`

Reply:
![image](https://user-images.githubusercontent.com/41823689/155393009-f90bd70a-f99c-458b-8f26-0f098ec8f412.png)

Reply:
Dobrze, że to zauważyłaś :)))